kolibri/utils/conf.py

```diff
@@ -64,9 +59,16 @@
 #: If a config file does not exist, we assume it's the first run
 config['FIRST_RUN'] = True
 
-# Open up the config file and overwrite defaults
-with open(conf_file, 'r') as kolibri_conf_file:
-    config.update(json.load(kolibri_conf_file))
+conf_file = os.path.join(KOLIBRI_HOME, "kolibri_settings.json")
+
+if not os.path.isfile(conf_file):
+    logger.info("Initialize kolibri_settings.json..")
+    with open(conf_file, 'w') as kolibri_conf_file:
+        json.dump(config, kolibri_conf_file)
```

Comment:
Looks like this code is identical to the contents of the `save` function -- could you call that instead?